### PR TITLE
48530: New Application Button

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: 

Preview: https://deploy-preview-362--replicated-docs.netlify.app/vendor/releases-creating-releases

### CODE REVIEW

**Review by paigecalvert — APPROVED** [review]
lgtm -- just added one suggestion

**Review by ajp-io — CHANGES_REQUESTED** [review]

**Comment by jonquil2002** [comment]
@paigecalvert  @ajp-io  Ready for another review pass. Good ideas, thanks!

**Review by ajp-io — APPROVED** [review]
@miaawong These docs should get merged when we ship the updated application dropdown and new create app button.

File: docs/vendor/releases-creating-releases.md [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
+1. Log in (or create a new team) on the vendor portal and do one of the following:
 
-  **Note:** If you are logging in with an existing account, you are not prompted to create a new application. Instead, click https://vendor.replicated.com/new-application to create a new application.
+    - If you are activating a new account: In the prompt, name your application and click **Create application**.
 
-  ![Create Application](/images/guides/kots/create-application.png)
+    ![Create Application](/images/guides/kots/create-application.png)
 
-  The Channels page opens and displays a list of your release channels, which are logical stacks for you to stage and promote releases to your customers.
+    - If you are logging in with an existing account: Click **New Application** or select an existing application.
```
Comment (ajp-io) [inline docs/vendor/releases-creating-releases.md:11]: We decided against a dedicated New Application button, so we moved this back into the dropdown. So creating a new app is in the dropdown again.

**Review by ajp-io — COMMENTED** [review]
Thanks for resurfacing this PR. We decided against a dedicated New Application button, so that part needs to change to mention how it's in the dropdown (as before). But if you think the rest of the changes here are worth making, then we can definitely get this approved and merged.

**Comment by jonquil2002** [comment]
Closing this PR because the feature was archived and the other non-feature changes are not needed due to another PR from last week.

**Comment by ajp-io** [comment]
@jonquil2002 One thing I liked in this PR was that instead of referencing a random link in the first step [here](https://docs.replicated.com/vendor/releases-creating-releases), we talk about how to find the New Application button. We. can make this change separately, but I would rather tell people to use the standard button instead of this random link. This link isn't accessible anywhere in the product—like you can't click something in the product to get to that. So I'd rather show people how they can do this in their normal workflow, rather than using a random link they have to bookmark.

**Pull request closed without merging**